### DISCUSSION TITLE
Lost my Insomnia projects after signing out

### DISCUSSION THREAD

**Discussion by wayn-amir** [Kong/insomnia#7110] [Help]
Hi,

I used Insomnia for the last many months.
Recently I had to switch my machine and downloaded a fresh copy of Insomnia.
I signed into my account.
I realized it blank out all my projects as it shows only Personal Project folder.
I went back to my old machine and noticed I was signed out from there as well.
I am really confused and don’t know which email account I had originally signed in.
The only thing I remember was my account name visible on top right corner on the window.
I would really appreciate if someone helps !

**Comment by subnetmarco** [comment]
Hi @wayn-amir - Make sure you sign-in using the same account. If you are sure that you are using the same account, you can always double check to pull your collections from the pull button in the dashboard.

If not, your collections are stored in the file system in the Insomnia folder.
